app/src/main/java/com/nextcloud/talk/chat/MessageInputFragment.kt

```diff
@@ -838,7 +838,8 @@ class MessageInputFragment : Fragment() {
                     mentionId.contains("@") ||
                     mentionId.startsWith("guest/") ||
```

Comment:
```suggestion
                    mentionId.contains("/")
```

And simplify your life? ;)